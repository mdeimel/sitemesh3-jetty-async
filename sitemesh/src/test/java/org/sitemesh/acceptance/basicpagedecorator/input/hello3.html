<title>Hello 3</title>

<p>A document that is not properly enclosed in html/body tags. Not a problem.</p>