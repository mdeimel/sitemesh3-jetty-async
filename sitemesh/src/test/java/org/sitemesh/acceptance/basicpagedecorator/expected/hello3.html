<!-- Common header -->
<html>
  <head>
    <title>Hello 3</title>
    <style type="text/css">
      /* common css */
      body { padding: 20px; }
    </style>
  </head>
  <body>

    <h1 style="text-align:center">Hello 3</h1>

    <!-- body -->
    <p>A document that is not properly enclosed in html/body tags. Not a problem.</p>
    <!-- end of body -->

  </body>
</html>
<!-- Common footer -->